Java/src/com/couchbase/litecore/C4ReplicatorListener.java

```diff
@@ -21,7 +21,7 @@
     void statusChanged(final C4Replicator replicator,
                        final C4ReplicatorStatus status, final Object context);
 
-    void documentError(final C4Replicator replicator, final boolean pushing,
+    void documentEnded(final C4Replicator replicator, final boolean pushing,
```

Comment:
Could you check if this updated? I remembered that LiteCore changed to something like below:

```
void onDocEnded(C4Replicator *repl,
                                bool pushing,
                                C4HeapString docID,
                                C4HeapString revID,
                                C4RevisionFlags flags,
                                C4Error error,
                                bool errorIsTransient,
                                void *context)
```